versioned_docs/version-stable/cadence/language/built-in-functions.mdx

```diff
@@ -37,15 +37,14 @@ fun revertibleRandom(): UInt64
 ```
 
 <Callout type="info">
-`unsafeRandom()` function currently behaves the same way as `revertibleRandom()`,
-and will be deprecated in the next Cadence release.
+`unsafeRandom()` function currently behaves the same way as `revertibleRandom()`.
+`unsafeRandom` is deprecated and will be removed in an upcoming release of Cadence.
 </Callout>
```

Comment:
Remove the whole callout, it's a duplicate of the one on the bottom for `unsafeRandom`